internal/encoding/parquet/parquet.go

```diff
@@ -80,25 +80,33 @@ func (i *iterator) Range(f func(int, []interface{}) bool, columns ...string) (in
 		}
 
 		var arr []interface{}
+		m := make(map[string]interface{})
```

Comment:
Preallocate this to number of columns. Also, this is going to be way slower to ingest.